File: goty-server/src/main/java/com/aleinin/goty/objectmapper/ObjectMapperConfiguration.java
package com.aleinin.goty.objectmapper;

import com.fasterxml.jackson.databind.ObjectMapper;
import com.fasterxml.jackson.databind.json.JsonMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
class ObjectMapperConfiguration {

    @Bean
    ObjectMapper objectMapper() {
        return JsonMapper.builder()
                .build();
    }
}

File: goty-server/src/main/java/com/aleinin/goty/objectmapper/ObjectMapperConfiguration.kt
package com.aleinin.goty.objectmapper

import com.fasterxml.jackson.databind.json.JsonMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class ObjectMapperConfiguration {

    @Bean
    fun objectMapper() = JsonMapper.builder().build()!!
}
